src/baseconverter.js

```diff
@@ -1,18 +1,30 @@
 'use strict';
 
+
 function normalize(digitArray) {
-  while (digitArray[0] === 0) {
-    digitArray.shift();
+  const length = digitArray.length;
```

Comment:
in 0.13 or in 0.12?
because it is used in 0.12, and RippleTrade is using 0.12, and divmod was slowest part of the code